packages/ui/src/components/form-input/index.ts

```diff
@@ -17,7 +17,17 @@ const FormInput = {
     checked: field.value,
     onCheckedChange: field.onChange
   })),
+  Select: withForm(Select, ({ field }) => ({
```

Comment:
Do we need these type casting? It makes the file look too complicated. Can we have `onSelectedChange` as `onChange`? So that it can be simplified to 

```js
Select: withForm(Select)
```

If not, we can have it like

```ts
Select: withForm(Select, ({ field }) => ({
  onSelectedChange: field.onChange
})),
```

Reply:
According to the [API](https://gist.github.com/abhinavrastogi-harness/984c6511de5f989d651cd7a7ef2e0c59), `onChange` has a different props interface, this is why I had to add `onSelectedChange`

Reply:
Originally, I've added it for the integration with `react-hook-form`, because it passes `onChange` callback with a different props interface to the component. I tested if I can use the default Select's onChange, and it works. Don't know how and why, but anyways - I removed `onSelectedChange`🤷‍♂️